src/lib/app.ts

```diff
@@ -30,18 +32,18 @@ module.exports = (opts) => {
   if (opts && opts.isProd) {
     // in production, use Bearer Strategy for auth
     passport.use(new BearerStrategy({
-      identityMetadata: 'https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration',
-      clientID: opts.authClientId,
       audience: opts.authAudience,
+      clientID: opts.authClientId,
+      identityMetadata: 'https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration',
```

Comment:
this should come from opts, and default to this value

Reply:
IMO we should move this concern to issue #17